Composer/packages/extension-client/src/hooks/usePublishApi.ts

```diff
@@ -58,6 +58,13 @@ export function usePublishApi() {
   function userShouldProvideTokens(): boolean {
     return window[ComposerGlobalName].userShouldProvideTokens();
   }
+  function getRequiredRecognizers(): {
+    projectId: string;
+    requiresLUIS: boolean;
+    requiresQNA: boolean;
+  }[] {
+    return window[ComposerGlobalName].getRequiredRecognizers();
```

Comment:
doesn't this need to pass the parameters through

Reply:
oh nevermind i see this is the return type.

Reply:
That function doesnt take params the bit of code 
```